components/homme/src/theta-l/element_ops.F90

```diff
@@ -433,13 +434,18 @@ subroutine set_thermostate(elem,ps,temperature,hvcoord)
   enddo
   elem%state%ps_v(:,:,nt)=ps
 
+
+  if (present(Q1)) then
+     call get_R_star(dp,Q1(:,:,:))  ! compute Rstar, store in dp
```

Comment:
"R" refers to a gas constant, correct? Might be good to clarify that with a comment describing what's happening here.

Reply:
fixed.